### PR TITLE
483 isRequestingUserAssociated

### PR DESCRIPTION
**Pull request by wbyoko**
this is for getConsolidatedCasesByCaseInteractor.js

### CODE REVIEW

File: shared/src/business/useCases/getConsolidatedCasesByCaseInteractor.js [hunk 1]
```diff
@@ -10,8 +10,34 @@ exports.getConsolidatedCasesByCaseInteractor = async ({
...
+    .getPersistenceGateway()
+    .getUserMappingByConsolidatedCases({
+      applicationContext,
+      consolidatedCases,
+    });
+
+  const isRequestingUserAssociatedMap = userMappings.reduce(
+    (acc, userMapping) => {
+      if (userMapping) {
+        acc[userMapping.sk] = true;
```
Comment (matthopson) [inline shared/src/business/useCases/getConsolidatedCasesByCaseInteractor.js:30]: Usually we call `stripInternalKeys` on dynamo results. Should we use the caseId instead here?

**Review by matthopson — APPROVED** [review]
I've got one question regarding evaluating an internal key outside of dynamo. I've seen some cases where we've not removed them - which I think was an accident, and was actually thinking about adding a task to go through and clean al of them up, but that would affect this as well. I think this is good to merge as-is until we figure that out though.

**Comment by matthopson** [comment]
@wbyoko before merging, please see convo in Slack regarding this. We may not need to move forward with this.

**Comment by wbyoko** [comment]
Pretty sure we can close it.

**Pull request closed without merging**